src/use-query.ts

```diff
@@ -73,6 +73,16 @@ export function useQuery<TResult, TError = ErrorDefault>(
   const refresh = () => store.refresh(entry.value)
   const refetch = () => store.refetch(entry.value)
 
+  // if passed by client
+  // & check if watchable
+  if (_options.enabled !== undefined && typeof options.enabled !== 'boolean') {
```

Comment:
since options merges the default value of `true`, this should also work, right?

```suggestion
  if (typeof options.enabled !== 'boolean') {
```